dist/main.js

```diff
@@ -0,0 +1,2068 @@
+(function(e, a) { for(var i in a) e[i] = a[i]; }(exports, /******/ (function(modules) { // webpackBootstrap
```

Comment:
Instead of checking these files in they can be apart of the `install` script and gitignored

Reply:
Thanks, I'm working on the build process first